src/client.coffee

```diff
@@ -192,10 +192,13 @@ class SlackClient
       thread_ts: envelope.message?.thread_ts
 
     if typeof message isnt "string"
-      @web.chat.postMessage(room, message.text, _.defaults(message, options))
+      messageOptions = _.defaults(message, options)
+      @robot.emit "postMessage", room, message.text, messageOptions
```

Comment:
if the goal is just to understand when `chat.postMessage` is called, then i don't think emitting on the `robot` is appropriate.

instead, i think the goal should be to turn on more logging, specifically for the `WebClient`. currently, there's a pattern established to set more options on the `RtmClient` using the `HUBOT_SLACK_RTM_CLIENT_OPTS` environment variable. so an alternative that i would favor is to introduce a `HUBOT_SLACK_WEB_CLIENT_OPTS` environment variable that can be used to set the `logLevel` to `"debug"` (and any of the other options).

what do you think?